core/data/data.go

```diff
@@ -8,15 +8,19 @@ import (
 	"database/sql"
 	"fmt"
 	"os"
+  "path/filepath"
 	"time"
-
+  
+  "github.com/owncast/owncast/config"
 	"github.com/owncast/owncast/utils"
 	log "github.com/sirupsen/logrus"
 )
 
 const (
 	schemaVersion = 0
-	backupFile    = "backup/owncastdb.bak"
+)
+var (
+	backupFile    = filepath.Join(config.BackupDirectory, "owncastdb.bak")
```

Comment:
Probably best just remove this since the `config.BackupDirectory` it's using is actually the default, so the customized directory isn't being used if you specify it on the command line.  Can just generate the full path when needed.
```
go run main.go --enableVerboseLogging -backupdir /tmp/testbackup
...
TRAC[2021-06-12T15:23:54-07:00] Backing up database to data/backup/owncastdb.bak
```

Reply:
@gabek Sorry about that! I hope the new commit fixes it.